specification/developerhub/resource-manager/Microsoft.DevHub/preview/2022-10-11-preview/workflow.json

```diff
@@ -0,0 +1,1008 @@
+          "$ref": "#/definitions/ArtifactGenerationProperties"
+        }
+      }
+    },
+    "WorkflowListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Workflow"
+          },
+          "description": "The list of workflows."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to the next set of workflow results.",
+          "readOnly": true
+        }
+      },
+      "description": "The response from List Workflows operation."
+    },
+    "PullRequestStatus": {
+      "type": "string",
+      "readOnly": true,
+      "enum": [
+        "unknown",
+        "submitted",
+        "merged",
+        "removed"
+      ],
+      "x-ms-enum": {
+        "name": "PullRequestStatus",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "unknown",
+            "description": "Pull Request state unknown."
+          },
+          {
+            "value": "submitted",
+            "description": "Pull Request submitted to repository."
+          },
+          {
+            "value": "merged",
+            "description": "Pull Request merged into repository."
+          },
+          {
+            "value": "merged",
+            "description": "Workflow no longer found within repository."
+          }
+        ]
+      },
+      "description": "The status of the Pull Request submitted against the users repository."
+    },
+    "DeploymentProperties": {
+      "type": "object",
+      "properties": {
+        "manifestType": {
+          "$ref": "#/definitions/ManifestType"
+        },
+        "kubeManifestLocations": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "helmChartPath": {
+          "type": "string",
+          "description": "Helm chart directory path in repository."
+        },
+        "helmValues": {
+          "type": "string",
+          "description": "Helm Values.yaml file location in repository."
+        },
+        "overrides": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "title": "Manifest override values.",
+          "description": "Manifest override values."
+        }
+      }
+    },
+    "ArtifactGenerationProperties": {
+      "type": "object",
+      "properties": {
+        "generationLanguage": {
+          "$ref": "#/definitions/GenerationLanguage"
+        },
+        "languageVersion": {
+          "type": "string",
+          "description": "The version of the language image used for execution in the generated dockerfile."
+        },
+        "builderVersion": {
+          "type": "string",
+          "description": "The version of the language image used for building the code in the generated dockerfile."
+        },
+        "port": {
+          "type": "string",
```

Comment:
>type": "string", [](http://example.com/codeflow?start=11&length=16)

should this be int type ?

Reply:
this was left as a string to allow future implementation of named ports